spark/interpreter/src/test/java/org/apache/zeppelin/spark/PySparkInterpreterTest.java

```diff
@@ -59,6 +59,9 @@ private static Properties getPySparkTestProperties() throws IOException {
     p.setProperty("zeppelin.dep.localrepo", tmpDir.newFolder().getAbsolutePath());
     p.setProperty("zeppelin.pyspark.useIPython", "false");
     p.setProperty("zeppelin.spark.test", "true");
+    if (null != System.getenv("SPARK_HOME")) {
```

Comment:
Where SPARK_HOME is set ? I don't see you set it in pom.xml

Reply:
`testing/downloadSpark.sh` sets it.

Reply:
This is what I concern, because in that case it is difficult for user to run unit test in local enviroment if they don't know the internal mechanism .